vm-image-spec.yaml

```diff
@@ -254,8 +254,8 @@ files:
           select
             case
               when pg_catalog.pg_is_in_recovery()
-              then pg_last_wal_replay_lsn()
-              else pg_current_wal_lsn()
+              then (pg_last_wal_replay_lsn() - '0/0')::FLOAT8
+              else (pg_current_wal_lsn() - '0/0')::FLOAT8
```

Comment:
Why these casts to float8?

Reply:
The story of this PR is working around Prometheus' limitations. Prometheus can only store float64. It somehow does not support any other data type, so we have to cast it.

That said, it's probably fine because a float64 should be able to store all integers between -2^53 and 2^53 exactly, and safekeepers have been implicitly casting LSNs to floats for a while now for their metrics.

Reply:
Ok. Maybe add a comment to explain that

Reply:
Added